manifests/broker.yml

```diff
@@ -10,6 +10,7 @@ meta:
     exodus_path: (( concat "secret/exodus/" meta.bosh_exodus_path "/cf" ))
     system_domain: (( vault meta.cf.exodus_path ":system_domain" ))
     api_url:    (( concat "https://api." meta.cf.system_domain ))
+    skip_ssl_validation: true
```

Comment:
move this meta line into metrics-emitter.yml, because the broker never actually talks _to_ the CF API, and doesn't care about this.

Reply:
also, make it a `param`